plugin/Record/RecordManagerReadForm.php

```diff
@@ -206,17 +223,19 @@ public function getRowByName($formName)
 	 *
 	 */
 
-	public function readFieldconfigByFormId($formId)
+	public function readFieldconfigByFormId(int $formId)
 	{
 		$prefix = $this->getTablePrefix();
 		$pWpDb = $this->getWpdb();
 
-		$sqlFields = "SELECT *
-			FROM {$prefix}oo_plugin_form_fieldconfig
-			WHERE `".esc_sql($this->getIdColumnMain())."` = ".esc_sql($formId)."
-			ORDER BY `order` ASC";
+		$sql = $pWpDb->prepare(
+			"SELECT *
+			FROM `{$prefix}oo_plugin_form_fieldconfig`
+			WHERE {$this->getIdColumnMain()} = %d",
+			$formId
+		);
```

Comment:
Hier fehlt ein ORDER BY von der Original Query: 

```suggestion
		$sql = $pWpDb->prepare(
			"SELECT *
			FROM `{$prefix}oo_plugin_form_fieldconfig`
			WHERE {$this->getIdColumnMain()} = %d ORDER BY `order` ASC",
			$formId
		);
```